### PR TITLE
[CIAPP-5225] Add information about flaky facets in Test Runs page.

### PR DESCRIPTION
**Pull request by alxch-**
### What does this PR do?

This PR adds documentation for a new feature: we are adding `new_flaky` and `known_flaky` attributes to test runs, and this PR adds ways to find relevant test runs for both facets.

### Motivation

We need to document that change so that customers are aware.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/guides/flaky_test_management.md [hunk 1]
```diff
@@ -35,13 +35,22 @@ If a flaky test has not failed in the past 30 days, it is automatically removed
 
 ## Watch for new flaky tests
 
+These are tests that exhibit flaky behavior and didn’t previously exist in the Flaky Tests table for the current branch or default branch of the repository.
+
+### In the Test Runs page
+
+1. In the facets list on the left sidebar, in the "Test" section, expand the "New Flaky" facet and check `true`
```
Comment (maycmlee) [inline content/en/continuous_integration/guides/flaky_test_management.md:42]:
```suggestion
1. Navigate to the [Test Runs][1] page.
2. In the facets list on the left sidebar, in the **Test** section, expand the **New Flaky** facet and check `true`.
```

File: content/en/continuous_integration/guides/flaky_test_management.md [hunk 2]
```diff
@@ -35,13 +35,22 @@ If a flaky test has not failed in the past 30 days, it is automatically removed
 
 ## Watch for new flaky tests
 
+These are tests that exhibit flaky behavior and didn’t previously exist in the Flaky Tests table for the current branch or default branch of the repository.
+
+### In the Test Runs page
```
Comment (maycmlee) [inline content/en/continuous_integration/guides/flaky_test_management.md:40]:
```suggestion
### Test Runs page
```

File: content/en/continuous_integration/guides/flaky_test_management.md [hunk 3]
```diff
@@ -51,10 +60,20 @@ Click on the **New Flaky** number and then click **Ignore flaky tests**.
...
+### In the Branches page
+
 1. On the Tests page, select the **Branches** view.
 
 2. Filter the table to see any branches, services, or commits of interest.
 
-3. The **Failed** column contains the number of failed tests and known flaky failed tests in the latest commit. Known flaky failed tests are tests that have flaky behavior on the current or default branch of the repository.
+3. The **Failed** column contains the number of failed tests and known flaky failed tests in the latest commit.
 
 {{< img src="ci/known-flaky-failed-tests.png" alt="CI Tests Branches view with a branch selected and a text box in the Failed column displaying 1 tests failed and 1 known flaky" style="width:100%;">}}
```
Comment (maycmlee) [inline content/en/continuous_integration/guides/flaky_test_management.md:79]:
```suggestion
{{< img src="ci/known-flaky-failed-tests.png" alt="CI Tests Branches view with a branch selected and a text box in the Failed column displaying 1 tests failed and 1 known flaky" style="width:100%;">}}

[1]: https://app.datadoghq.com/ci/test-runs
[2]: https://app.datadoghq.com/ci/test-services
```

File: content/en/continuous_integration/guides/flaky_test_management.md [hunk 4]
```diff
@@ -35,13 +35,22 @@ If a flaky test has not failed in the past 30 days, it is automatically removed
 
 ## Watch for new flaky tests
 
+These are tests that exhibit flaky behavior and didn’t previously exist in the Flaky Tests table for the current branch or default branch of the repository.
+
+### In the Test Runs page
+
+1. In the facets list on the left sidebar, in the "Test" section, expand the "New Flaky" facet and check `true`
+2. This displays all test runs that exhibited flakiness behavior for the first time as per the definition above.
```
Comment (maycmlee) [inline content/en/continuous_integration/guides/flaky_test_management.md:43]:
```suggestion
All test runs that exhibited flakiness behavior for the first time as per the definition above are displayed.
```

File: content/en/continuous_integration/guides/flaky_test_management.md [hunk 5]
```diff
@@ -35,13 +35,22 @@ If a flaky test has not failed in the past 30 days, it is automatically removed
...
+
+### In the Branches page
```
Comment (maycmlee) [inline content/en/continuous_integration/guides/flaky_test_management.md:45]:
```suggestion
### Branches page
```

**Review by maycmlee — CHANGES_REQUESTED** [review]
Left some feedback! Also added a couple of links to the pages in the app.

File: content/en/continuous_integration/guides/flaky_test_management.md (hunk 3 above)
Comment (alxch-) [reply content/en/continuous_integration/guides/flaky_test_management.md:79]: should the app URL differ depending on the chosen datacenter?

**Review by juan-fernandez — APPROVED** [review]

File: content/en/continuous_integration/guides/flaky_test_management.md [hunk 6]
```diff
@@ -35,13 +35,21 @@ If a flaky test has not failed in the past 30 days, it is automatically removed
...
 
-1. On the Tests page, select the **Branches** view.
+These are tests that exhibit flaky behavior and didn’t previously exist in the Flaky Tests table for the current branch or default branch of the repository.
 
-2. Filter the table to see branches, services, or commits of interest to you.
+### Test Runs page
+
+1. Navigate to the [Test Runs][1] page.
+2. In the facets list on the left sidebar, expand the **New Flaky** facet in the **Test** section, and check `true`.
+All test runs that exhibited flakiness behavior for the first time as per the definition above are displayed.
```
Comment (maycmlee) [inline content/en/continuous_integration/guides/flaky_test_management.md:44]:
```suggestion
All test runs that exhibited flakey behavior for the first time as per the definition above are displayed.
```

**Review by maycmlee — APPROVED** [review]

**Pull request merged**